source/reference/cutover-process.txt

```diff
@@ -43,15 +43,40 @@ Steps
   
       - ``canCommit`` is ``true``. 
 
+      The following example returns the status of the synchronization process.
```

Comment:
[Suggestion]

Maybe we can reorder the preceding bullets so the fields appear in the same order as the code block.

Reply:
I like this!